test/validation/api/pool/pool.c

```diff
@@ -1121,6 +1130,356 @@ static void pool_test_timeout_pool_statistics(void)
 	pool_test_pool_statistics(ODP_POOL_TIMEOUT);
 }
 
+static void pool_ext_init_packet_pool_param(odp_pool_ext_param_t *param)
+{
+	odp_pool_ext_capability_t capa;
+	uint32_t head_offset, head_align, trailer_size;
+	odp_pool_type_t type = ODP_POOL_PACKET;
+	uint32_t num_buf = EXT_NUM_BUF;
+	uint32_t buf_size = EXT_BUF_SIZE;
+	uint32_t uarea_size = EXT_UAREA_SIZE;
+	uint32_t headroom = EXT_HEADROOM;
+	uint32_t app_hdr_size = EXT_HDR_SIZE;
+
+	CU_ASSERT_FATAL(odp_pool_ext_capability(type, &capa) == 0);
+
+	odp_pool_ext_param_init(type, param);
```

Comment:
Fixed in v15. Added new test case for it, and changed API to define 0 as default value for app_hdr_size and uarea_size. I didn't change headroom documentation as it would differ from normal pool (implementation) then.